index.html

```diff
@@ -176,7 +176,7 @@ <h2><i>When The Sun Goes Down</i></h2>
 				</div> 
                 </div><!--/.col-xs-6.col-lg-4-->
             
-            <div class="album col-xs-6 col-lg-4">
+            <div class="album col-xs-6 col-lg-4" id="float-none">
```

Comment:
change `float-none` to class.  Similarly for others.

Reply:
changing it to class doesn't work because other classes overwrite the properties.

Reply:
Although this should not happen. I guess, you may be missing something. Anyways try using `!important` directive of CSS if it other classes are overriding the property.

Reply:
Ill make this another issue.